web/components/application/hacker/pages/Page.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Page extends React.Component {
```

Comment:
Awesome abstraction 🙏 do you think we could move this to `web/components/application` so we don't have to duplicate it when doing the volunteer application?